### PR TITLE
#1 #33 - seeding database and special price

### LINKED ISSUE #1: Create test data
Implement code which will populate a database with test data. Code may run before each run, code certainly has to run before integration tests. 

**!!! Code HAVE TO be save in a sense that it will be hard to erase production database**

Possible solutions:
write SQL script
write c# code

### LINKED ISSUE #33: Special price
Add special price to db model.

### PR DESCRIPTION
**Pull request by Erunno**
I've created a new project to the solution - it is an app which will seed database - it is very safe since it explicitly seeds Sqlite DB

Also, I've added a new "DB table" `SpecialPrice` - an owner of a bottle can decide that some users will drink for different prices than others.   

closes #1
closes #33

### CODE REVIEW

File: WhiskyMan.Backend/WhiskyMan.sln [hunk 1]
```diff
@@ -25,7 +25,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WhiskyMan.Repositories.Test
 EndProject
```
Comment (Erunno) [inline WhiskyMan.Backend/WhiskyMan.sln:25]: changes in this file are OK - I've added a new project

**Comment by Erunno** [comment]
I wrote [wiki page](https://github.com/Erunno/WhiskyMan/wiki/Seeding-local-database). Verify it, please.

File: WhiskyMan.Backend/WhiskyMan.API/appsettings.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "AppSettings": {
-    "Token": "super secure key <replace in production>"
+    "Token": "super ultra secure key"
```
Comment (Jokekill) [inline WhiskyMan.Backend/WhiskyMan.API/appsettings.json:3]: Is this really secure?

File: WhiskyMan.Backend/WhiskyMan.Repositories/Mapping/UserMaps.cs [hunk 3]
```diff
@@ -1,7 +1,6 @@
-﻿using AutoMapper;
+using AutoMapper;
```
Comment (Jokekill) [inline WhiskyMan.Backend/WhiskyMan.Repositories/Mapping/UserMaps.cs:1]: this didnt change or am i going crazy?

File: WhiskyMan.Backend/seed_database.bat [hunk 4]
```diff
@@ -0,0 +1 @@
+dotnet run --project WhiskyMan.DatabaseSeedApp WhiskyMan.API\whiskyMan.db
```
Comment (Jokekill) [inline WhiskyMan.Backend/seed_database.bat:1]: Add newline at the end of the line. just a convention

**Review by Jokekill — COMMENTED** [review]
It looks all good. Only the 'estetic' changes are requierd

File: WhiskyMan.Backend/WhiskyMan.API/appsettings.json (hunk 2 above)
Comment (Erunno) [reply WhiskyMan.Backend/WhiskyMan.API/appsettings.json:3]: Only for develop

File: WhiskyMan.Backend/WhiskyMan.Repositories/Mapping/UserMaps.cs (hunk 3 above)
Comment (Erunno) [reply WhiskyMan.Backend/WhiskyMan.Repositories/Mapping/UserMaps.cs:1]: Weird... github... probably ok

**Comment by Erunno** [comment]
@Jokekill issues solved. Merge it if you think it's OK.

**Review by Jokekill — APPROVED** [review]

**Comment by Jokekill** [comment]
All issues are gone

**Pull request merged**